Quantum-35/Ride_My_Way_Challenge2
# Ft develop get all rides #158497612

**What does this PR do?**
Finishes the endpoint that the **Registered** and and **Logged** in user can get all the available rides
Uses JWT tokens to restict access to this endpoint by unregistered users or users who have not logged in

**Description of Task to be completed**
Implement Pep8

**How should I test it?**
Clone this repository to a local folder
Navigate to the cloned repo and Run:
On Linux or macos terminal

    python3 -m venv venv
    . venv/bin/activate

Or on Windows cmd::

    py -3 -m venv venv
    venv\Scripts\activate.bat

Install the dependencies::

    pip install -r requirements.txt 
    *Then git  ft-develop get all rides #158497612
Run the development server:
  ```
  $ python run.py
  On postman :
 send **Post** request  to **http://127.0.0.1:5000/api/v1/auth/signup** to sign up
send **Post** request  to **http://127.0.0.1:5000/api/v1/auth/login** to login and automatically generates access token
send **Get** request  to **http://127.0.0.1:5000/api/v1/rides/** to get all available rides and make sure you add the  access token to the header
  ```
**To run test with coverage**
pytest --cov-report term-missing --cov=app
**What PT stories?**
https://www.pivotaltracker.com/n/projects/2179783
